chain/sub/incoming.go

```diff
@@ -443,19 +444,12 @@ func (bv *BlockValidator) checkPowerAndGetWorkerKey(ctx context.Context, bh *typ
 			return address.Undef, err
 		}
 
-		blk, err := bv.chain.Blockstore().Get(act.Head)
-		if err != nil {
-			return address.Undef, err
-		}
-		aso := blk.RawData()
-
-		var mst miner.State
-		err = mst.UnmarshalCBOR(bytes.NewReader(aso))
+		mst, err := miner.Load(bv.chain.Store(ctx), act)
```

Comment:
This is the new access pattern. Use the actor's load function to load an abstraction over the actor, then call methods on the abstraction.